hydra-cluster/exe/hydra-cluster/Main.hs

```
@@ -45,22 +57,23 @@ run options =
 
   withRunningCardanoNode tracer workDir network action =
     findRunningCardanoNode (contramap FromCardanoNode tracer) workDir network >>= \case
-      Just node ->
-        action node
+      Just (blockTime, backend) ->
+        action blockTime backend
       Nothing -> do
         when (useMithril == UseMithril) $ do
-          removeDirectoryRecursive $ workDir </> "db"
+          removeDirectoryRecursive (workDir </> "db") `catch` (\(_ :: SomeException) -> pure ())
```

why do we ignore the exception?

---

This was a simple _life improvement_ since the only way to get exception is if the folder is not there. But in any case mithril will download the db so this should not stop the test in case the folder was empty to start with.